companion/index.js

```diff
@@ -134,6 +135,11 @@ function sendDataToInflux(data) {
         api_key = "UBQpWptj9HaBJVAVEDOZ14aQoNh7EpTK9zccvBTa"
     }
 
+    try {api_key = JSON.parse(settingsStorage.getItem('api_key')).name}
+    catch (error) {
+        api_key = "UBQpWptj9HaBJVAVEDOZ14aQoNh7EpTK9zccvBTa"
```

Comment:
Can you start a security control document for Budslab which has a list of keys that exist in public code. I think its good to start this practice.